### PR TITLE
Nzpmc 030/single question display

### PR DESCRIPTION
**Pull request by matthewaptaylor**
**Issue**

NZPMC-030 and NZPMC-032

Sidebar displayed in the wrong position when window refreshed, question image sizes are too small, question number not displayed correctly, option box sizes

**Solution**

Overwrote Vuetify code trying to do its own thing, for our own custom use, updated timer to work in hours, used flex stretch property to make everything the same height and used the index of everything in the sidebar to update the question number for each question view.

**Risk**

None known.

**Reviewers**

@scorpionknifes @Will-ZYS @Kingston802 @ruby-oujo

### CODE REVIEW

File: src/components/Topbar.vue [hunk 1]
```diff
@@ -41,13 +41,17 @@ export default {
...
         },
         formattedTimeRemaining() {
-            const minutes = Math.floor(this.timeRemaining / 60)
+            const hours = Math.floor(this.timeRemaining / 3600)
+            const minutes = Math.floor((this.timeRemaining % 3600) / 60)
             let seconds = this.timeRemaining % 60
 
             if (seconds < 10) {
                 seconds = `0${seconds}`
             }
```
Comment (Will-ZYS) [inline src/components/Topbar.vue:50]: might wanna do the same for hours and mins as well, just to be consistent

Comment (matthewaptaylor) [reply src/components/Topbar.vue:56]: :+1:

**Review by Will-ZYS — APPROVED** [review]
LGTM

**Review by scorpionknifes — APPROVED** [review]
LGTM

**Pull request merged**